assembl/indexing/utils.py

```diff
@@ -120,6 +120,9 @@ def get_data(content):
                      'creator_id', 'sentiment_counts'):
             data[attr] = getattr(content, attr)
 
+        data['idea_id'] = [link.idea_id
```

Comment:
A message can be associated to several ideas, so idea_id would not be the best name, rather something like ideas_ids, no?

Reply:
Yeah, on the db model side. In v2, a message can only be associated to one idea though. The purpose of indexing idea_id here is to generate an url to the post, so I really only need one idea in the end.

Reply:
Maybe it can be only associated to one idea for now in v2, but that is temporary, as harvesting is a core feature of assembl, it will arrive sooner or later in v2